tmail-backend/webadmin/webadmin-team-mailboxes/src/main/java/com/linagora/tmail/webadmin/TeamMailboxManagementRoutes.java

```diff
@@ -62,13 +63,17 @@ public String getUsername() {
     private final TeamMailboxRepository teamMailboxRepository;
     private final DomainList domainList;
     private final JsonTransformer jsonTransformer;
+    private final TeamMailboxCallbackUtils teamMailboxCallback;
 
     @Inject
     public TeamMailboxManagementRoutes(TeamMailboxRepository teamMailboxRepository,
-                                       DomainList domainList, JsonTransformer jsonTransformer) {
+                                       DomainList domainList,
+                                       JsonTransformer jsonTransformer,
+                                       TeamMailboxCallbackUtils teamMailboxCallback) {
```

Comment:
No still not. The callbacks needs to be part of TeamMailboxesRepository. As a caller I should not need to worry and even know about it.